core/src/env/test_env.rs

```diff
@@ -17,8 +17,7 @@
 use super::*;
 use crate::{
     env::{
-        self,
-        srml,
+        EnvTypes,
```

Comment:
Is this already passing the formatting tests?

Reply:
Fixed